Packages/Tracking Preview/PhysicsHands/Runtime/PhysicsHand.cs

```diff
@@ -229,6 +273,44 @@ private void Start()
 
             gameObject.SetActive(false);
             _physicsHand.gameObject.SetActive(false);
+
+            InitJobs();
+        }
+
+        private void InitJobs()
+        {
+#if UNITY_2022_3_OR_NEWER
+            _overlapsJob = new PhysMultiOverlapJob()
+            {
+                point0 = new NativeArray<Vector3>(_physicsHand.jointBones.Length * 2, Allocator.Persistent),
```

Comment:
Can we make a local var of `_physicsHand.jointBones.Length * 2` rather than recalculate for each element?